org.eclim.jdt/java/org/eclim/plugin/jdt/command/doc/GetElementDocCommand.java

```diff
@@ -112,6 +113,10 @@ public Object execute(CommandLine commandLine)
       return null;
     }
 
+    if(returnHTML(commandLine)){
```

Comment:
After changing the arg to a `NOARG`, you can simply use the following here and remove the method you added:

    if (commandLine.hasOption(Options.HTML_OPTION)){
      ...